### ISSUE TITLE
Display of NFT ordinal and token ID

### ISSUE THREAD

**Issue by elv-serban** [eluv-io/elv-media-wallet#3]
## Overview

The NFT has a token ID and an ordinal (the ordinal is specific to the ElvTradableLocal contract and not part of the ERC721/1155 standard).  We need to show this information sensibly in the wallet in a way that both correctly identifies the NFT per contract and at the same time shows a good visual representation of the cardinality of this NFT in a series.

## Proposal

The **description** of the NFT should show both:

- Token ID (which is a big number)
  - Optionally if we detect this is a vanity token 
- Token ordinal - unaltered ordinal starting at 0 (as read from the contract)

The **NFT frame** should show the cardinality of this token starting at 1.

In other words if this is the first NFT of 500 - show `1 / 500`  (and in description the ordinal shows `0`)
If this is the first NFT of a series of one - show `1/1` (and in the description the ordinal shows `0`)

**Comment by camgirl** [comment]
Very good. Only comment is that the original reason to change this was to resolve the discrepancy between the token ordinal and the default token id, but this approach is purely 'correct' and we will go with it.

**Comment by elv-serban** [comment]
Verified
